kolibri/plugins/user/assets/src/views/AuthBase.vue

```diff
@@ -198,10 +198,11 @@
         return this.facilityConfig.learner_can_sign_up;
       },
       signUpPage() {
+        let signUpRoute = this.$router.getRoute(ComponentMap.SIGN_UP);
```

Comment:
This can be a `const` instead of a `let` as you are not reassigning the value anywhere in the scope.